### DISCUSSION TITLE
Snails & Slugs

### DISCUSSION THREAD

**Discussion by chris-bsnake** [BattlesnakeOfficial/feedback#30] [General]
> Originally posted by community member **altersaddle** on Slack

Sort of a variant of the Constrictor mode.  Your snake moves as normal/expected, but squares vacated by snakes are set to be hazard squares on the board (taking rules from Royale mode for hazards effect, but appearing behind snake paths rather than from out edges of the board).  In addition, when a snake dies, it's entire body goes away but hazard squares are left behind.

Game Mode Name Ideas:
* Hazard Trail
* Chemtrail
* Sauce Tracks

**Comment by chris-bsnake** [comment]
👍

**Comment by Nettogrof** [comment]
👍

**Comment by mamamia5x** [comment]
👍
